default_launch.json

```diff
@@ -8,7 +8,13 @@
     "dimensions": {
       "xMeters": 5000,
       "zMeters": 5000
-    }
+    },
+    "legacy_flags": [
+      {
+        "name": "bridge_qos_max_timeout",
+        "value": "0"
```

Comment:
I'm not sure about having this committed. It's more of a local debugging tool. If we need to add it in I'd suggest a different deployment config for it.

Reply:
Discussed offline. Putting this in the local launch config but not the cloud one is fine.